aspnetcore/security/authentication/mfa.md

```diff
@@ -0,0 +1,562 @@
+---
+title: Multi-factor authentication
+author: damienbod
+description: Multi-factor authentication
+monikerRange: '>= aspnetcore-3.1'
+ms.author: rick-anderson
+ms.custom: mvc
+ms.date: 02/25/2020
+no-loc: [Identity]
+uid: security/authentication/mfa
+---
+# Multi-factor authentication
+
+By [Damien Bowden](https://github.com/damienbod)
+
+Multi-factor authentication (MFA) is a process where a user is requested during a sign-in event for additional forms of identification. This prompt could be to enter a code on their cellphone, use a FIDO2 key or to provide a fingerprint scan. When you require a second form of authentication, security is increased as this additional factor isn't something that's easy for an attacker to obtain or duplicate.
+
+This article covers the following:
+
+* What is MFA and what MFA flows are recommended
+* Configure MFA for administration pages using ASP.NET Core Identity
+* Send MFA signin requirement to OpenID Connect server 
+* Force ASP.NET Core OpenID Connect client to require MFA
+
+## MFA, 2FA
+
+MFA requires at least two or more types of proof for an identity like something you know, possess, or inherit for the user to authenticate.
```

Comment:
What's meant by inherit here?

Reply:
Updated text:

MFA requires at least two or more types of proof for an identity like something you know, something you possess, or biometric validation for the user to authenticate.